code/modules/detective_work/evidence.dm

```diff
@@ -80,3 +80,175 @@
+	desc = "Used to store fiber evidence for forensic examianation."
+	icon_state = "fiberbag"
+
+/obj/item/sample/print
+	name = "fingerprint card"
+	desc = "Preserves fingerprints."
+	icon = 'icons/obj/card.dmi'
+	icon_state = "fingerprint0"
+	item_state = "paper"
+	var/used
+
+/obj/item/sample/print/update_icon_state()
+	. = ..()
+	icon_state = "fingerprint0"
+	if(used)
+		icon_state = "fingerprint1"
+
+
+/obj/item/sample/print/activate_self(mob/user)
+	if(..() || length(evidence) || !ishuman(user))
+		return FINISH_ATTACK
+
+	var/mob/living/carbon/human/H = user
+	if(H.gloves)
+		to_chat(user, "<span class='warning'>Take [H.gloves] off first.</span>")
+		return FINISH_ATTACK
+
+	to_chat(user, "<span class='notice'>You press your fingertips firmly against the card.</span>")
+	var/fullprint = H.get_full_print()
+	evidence[fullprint] = fullprint
+	name = "[initial(name)] ([H])"
+	used = TRUE
+	update_appearance(UPDATE_ICON_STATE)
+
+/obj/item/sample/print/interact_with_atom(atom/target, mob/living/user, list/modifiers)
+	if(!ishuman(target))
+		return ..()
+
+	if(length(evidence))
+		return ITEM_INTERACT_COMPLETE
+
+	var/mob/living/carbon/human/H = target
+
+	if(H.gloves)
+		to_chat(user, "<span class='warning'>[H] is wearing gloves.</span>")
+		return ITEM_INTERACT_COMPLETE
+
+	if(user != H && !IS_HORIZONTAL(H))
+		user.visible_message("<span class='danger'>[user] tried to fingerprint [H], but he resists.</span>")
+		return ITEM_INTERACT_COMPLETE
+
+	if(user.zone_selected == "r_hand" || user.zone_selected == "l_hand")
+		var/has_hand = (H.has_organ("r_hand") || H.has_organ("l_hand"))
+		if(!has_hand)
+			to_chat(user, "<span class='warning'>[H] has no hands!</span>")
+			return ITEM_INTERACT_COMPLETE
+		if(!do_after(user, 2 SECONDS, target = user))
+			return ITEM_INTERACT_COMPLETE
+
+		user.visible_message("<span class='notice'>[user] makes a copy of [H]'s fingerprints'.</span>")
+		var/fullprint = H.get_full_print()
+		evidence[fullprint] = fullprint
+		copy_evidence(src)
+		name = ("[initial(name)] ([H])")
+		used = TRUE
+		update_appearance(UPDATE_ICON_STATE)
+	return ITEM_INTERACT_COMPLETE
+
+/obj/item/sample/print/copy_evidence(atom/supplied)
+	if(length(supplied.fingerprints))
+		for(var/print in supplied.fingerprints)
+			evidence[print] = supplied.fingerprints[print]
+		supplied.fingerprints.Cut()
+
+/obj/item/forensics/sample_kit
+	name = "fiber collection kit"
+	desc = "Magnifying glass and tweezers. Used to lift fabric fibers. Use on harm intent to collect samples and not interact with objects."
+	icon_state = "m_glass"
+	w_class = WEIGHT_CLASS_SMALL
+	///naming for individual evidence items
+	var/evidence_type = "fibers"
+	var/evidence_path = /obj/item/sample/fibers
+
+/obj/item/forensics/sample_kit/proc/can_take_sample(mob/user, atom/supplied)
+	return length(supplied.suit_fibers)
+
+/obj/item/forensics/sample_kit/proc/take_sample(mob/user, atom/supplied)
+	var/obj/item/sample/S = new evidence_path(get_turf(user), supplied)
```

Comment:
You probably want to do something like this to make sure admins dont do something like spawning a singulo in them or something idk
```suggestion
	if(!ispath(evidence_path, /obj/item/sample)
		return
	var/obj/item/sample/S = new evidence_path(get_turf(user), supplied)
```